src/EVM/Expr.hs

```diff
@@ -978,6 +978,11 @@ evalProp prop =
     go (PNeg (PBool b)) = PBool (Prelude.not b)
     go (PNeg (PNeg a)) = a
 
+    -- solc specific stuff
+    go (PNeg (PEq (IsZero (IsZero a)) (Lit 0))) = PGT a (Lit 0)
```

Comment:
We could instead add the rewrite:
```
PEq (IsZero (IsZero a)) (Lit 0) = IsZero a
```

Then `PNeg (PEq (IsZero (IsZero a)) (Lit 0))) ` would become `PNeg (IsZero a)` which we could then rewrite into `PGT a (Lit 0)` with the rule:

```
go (PNeg (IsZero a)) = `PGT a (Lit 0)`
```

That would be more generic I think?